testsuite/MDAnalysisTests/coordinates/test_chainreader.py

```diff
@@ -115,6 +122,40 @@ def test_write_dcd(self, universe, tmpdir):
                 self.prec,
                 err_msg="Coordinates disagree at frame {0:d}".format(
                     ts_orig.frame))       
+    
+    def test_transform_iteration(self, universe, transformed):
```

Comment:
these tests look good, I'd add in a test for `rewind()` too, it should restore the first frame.